mgmt/rest/v2/config.go

```diff
@@ -33,17 +33,51 @@ type PolicyTable cpolicy.RuleTable
 
 type PolicyTableSlice []cpolicy.RuleTable
 
-// cdata.ConfigDataNode implements it's own UnmarshalJSON
 type PluginConfigItem struct {
 	cdata.ConfigDataNode
 }
 
+// PluginConfigResponse represents the response of a plugin config items.
+//
+// swagger:response PluginConfigResponse
+type PluginConfigResponse struct {
```

Comment:
@candysmurf It will be used in future PR as I understand? Because it is not used anywhere now.